sierra_adapter/sierra_item_merger/src/main/scala/uk/ac/wellcome/platform/sierra_item_merger/dynamo/MergedSierraRecordDao.scala

```diff
@@ -0,0 +1,55 @@
+package uk.ac.wellcome.platform.sierra_item_merger.dynamo
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.google.inject.Inject
+import com.gu.scanamo.{Scanamo, Table}
+import com.gu.scanamo.ops.ScanamoOps
+import com.gu.scanamo.syntax._
+import com.twitter.inject.Logging
+import uk.ac.wellcome.models.MergedSierraRecord
+import uk.ac.wellcome.dynamo._
+import uk.ac.wellcome.models.aws.DynamoConfig
+
+import scala.concurrent.Future
+import uk.ac.wellcome.utils.GlobalExecutionContext.context
+
+class MergedSierraRecordDao @Inject()(dynamoDbClient: AmazonDynamoDB,
```

Comment:
Very nice! Maybe we could put this straight into the sierra_adapter lib? I don’t see anything here which is item-merger specific, and it would allow us to tidy up the bib merger as well.

Reply:
Agreed. I think the work to use this in the bib merger is another PR though. Will move this into the sierra lib here though.

Reply:
I think we should definitely use it in the bib merger, but I would move it there as part of a refactoring patch on the bib merger, not on this patch

Reply:
I thought it might like to land there immediately because it was introduced in this patch, rather than being an edit to existing code, but I’m not massively fussed.